wom/src/test/scala/wdl4s/wdl/wom/WdlScatterWomSpec.scala

```diff
@@ -176,7 +176,7 @@ class WdlScatterWomSpec extends FlatSpec with Matchers {
 
       // Find the inputs:
       val inputNodes: Set[GraphInputNode] = workflowGraph.nodes.filterByType[GraphInputNode]
-      inputNodes.map {_.name} should be(Set("x", "foo.j"))
+      inputNodes.map {_.name} should be(Set("foo.j"))
```

Comment:
Just checking: this change is because `x` is a `DeclaredInputDefinition` and no longer generates a `GraphInputNode`?

Reply:
Yep